README.md

```diff
@@ -1 +1,24 @@
-# My-Portfolio
\ No newline at end of file
+# Project Name
+> My Professional Portfolio
+
+## Built With
+- HTML & CSS
+- Linters
+- .gitignore
+- Package.json
+
```

Comment:
[ ] kindly add a `Getting Started` section to your README file, This section should contain instructions on setting up your project locally. Example :Prerequisites :HTML, CSS , .. [Check this README Template](https://github.com/microverseinc/readme-template/blob/master/README.md)